lib/features/base/reloadable/reloadable_controller.dart

```diff
@@ -157,7 +158,9 @@ abstract class ReloadableController extends BaseController {
       ]);
       _authorizationInterceptors.clear();
       _authorizationIsolateInterceptors.clear();
+      FcmController.instance.deleteFcmToken();
```

Comment:
What happens if FCM is not available?

Reply:
Attention should be paid to the logout with `OIDC`